contracts/deploy/l1/02_deploy_Rollup.ts

```diff
@@ -6,25 +6,35 @@ import { DeployFunction } from "hardhat-deploy/types";
 
 import { deployUUPSProxiedContract, getProxyName } from "../utils";
 
-const CLIENT_SBIN_DIR = "../clients/geth/specular/sbin";
+const CLIENT_SBIN_DIR = `${__dirname}/../../../clients/geth/specular/sbin`;
 
 const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   // Calculate initial VM hash
   const execPromise = util.promisify(exec);
-  let initialVMHash = "";
+
+  const ls = await execPromise(`ls ${CLIENT_SBIN_DIR}`);
+  console.log({ ls });
+
+  await execPromise(
+    `chmod +x ${path.join(CLIENT_SBIN_DIR, "export_genesis.sh")}`
+  );
+  let initialVMHash;
   try {
-    const { stdout } = await execPromise(
-      path.join(CLIENT_SBIN_DIR, "export_genesis.sh")
+    const genesis = await execPromise(
+      `bash ${path.join(CLIENT_SBIN_DIR, "export_genesis.sh")}`
     );
-    initialVMHash = (JSON.parse(stdout).root || "") as string;
+
+    console.log({ genesis });
```

Comment:
either remove debug stuff or make it more readable

Reply:
Debug stuff removed